post_install.sh

```diff
@@ -1,18 +1,23 @@
 #!/bin/sh
-# Create user and set up the directorys
-pw useradd -n sickchill -c "Sickchill" -s /sbin/nologin -w no
-fetch https://github.com/SickChill/SickChill/archive/master.tar.gz
-tar xfz master.tar.gz -C /usr/local/
-mv /usr/local/SickChill-master /usr/local/sickchill
-chown -R sickchill:sickchill /usr/local/sickchill
-mkdir /usr/local/etc/rc.d
 
-# Enable
-sysrc -f /etc/rc.conf sickchill_enable="YES"
+echo "Create location for sickchill"
+mkdir /usr/local/app
+
+echo "Download sickchill repo"
+git clone https://github.com/SickChill/SickChill.git /usr/local/app/sickchill
+
+echo "Install dependencies"
+pip install -r /usr/local/app/sickchill/requirements.txt
 
-# Copy the init script where it's useful
-cp /usr/local/sickchill/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill
-chmod +x /usr/local/etc/rc.d/sickchill
+echo "Create sickchill user"
+pw user add sickchill -c "Sickchill" -d /nonexistent -s /usr/bin/nologin -w no
+chown -R sickchill:sickchill /usr/local/app/sickchill
+
+echo "Installing SickChill service"
+mkdir /usr/local/etc/rc.d
+cp /usr/local/app/sickchill/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill
```

Comment:
I think this line needs to be:

`cp /usr/local/app/sickchill/contrib/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill`

Reply:
Can you make the PR?

Reply:
Fixed